backend/src/dao/TeamsDao.ts

```diff
@@ -27,9 +26,29 @@ export default class TeamsDao {
     );
   }
 
+  static async getTeam(id: string): Promise<Team | null> {
+    const teamRef = await teamCollection.doc(id).get();
+    const team = teamRef.data();
+    if (!team) return null;
+    return {
+      uuid: team.uuid,
+      name: team.name,
+      members: await Promise.all(
+        team.members.map((ref) =>
+          ref.get().then((doc) => doc.data() as IdolMember)
+        )
+      ),
+      leaders: await Promise.all(
+        team.leaders.map((ref) =>
+          ref.get().then((doc) => doc.data() as IdolMember)
+        )
+      )
+    };
+  }
+
   static async setTeam(team: Team): Promise<Team> {
     const teamRef: DBTeam = {
-      uuid: team.uuid ? team.uuid : uuidv4(),
+      uuid: team.uuid,
```

Comment:
why this change?

Reply:
I think originally in the `updateTeamMembers` function, it checked if `team.uuid` was empty and generated a new one if it was. That way when `setTeam` was called, `team.uuid` would always be set already. I think that caused some linter error though since doing so updated the parameter to the function so the function now makes a copy of the team. So this line should actually probably. be changed back.